homeassistant/components/google_drive/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
@balloob do you have a recommendation for how we'll present this integration from a user perspective? We previously talked about having everything in `google` but i know we're moving towards moving related integrations at the UI view instead.

(Plus there are big auth differences since calendar uses device auth, though this integration has the right long term approach using web auth. Users can likely share creds web auth with nest so we could also merge there if we want)

Reply:
Integrations v2 seems like a good solution to keep these all separate.

Reply:
Is that in proposal state or something real?